### DISCUSSION TITLE
How to cite your work

### DISCUSSION THREAD

**Discussion by ecleipteon** [jonasthewolf/gsn2x#455] [General]
Hi Jonas, thanks again for your hard work in building gsn2x. I am a researcher (academia) and we have recently done some work on the automatic generation of assurance case fragments. 

We instrumented our software to output the cases according the syntax you defined for gsn2x (in YAML), and we use gsn2x to  visualise the cases. We are currently finalising a paper on our work, where we mention the use of gsn2x to visualise cases and include a citation linking to your GitHub repository. I was wondering, is there any document (e.g., paper, white-paper, etc.) you would prefer us to cite instead of (or in addition to) the repository?

Thanks!

**Comment by jonasthewolf** [comment]
Hi @ecleipteon 

thank you very much for citing gsn2x in your paper. I would love to read it, once it's published.
A link to the GitHub page with my name is absolutely sufficient. 
Please do not cite my current employer because they are completely unaffiliated.

Btw. I will be in York on the [SSS'25 ](https://scsc.uk/e1099) giving a short introductory workshop on gsn2x.

Best regards,

Jonas Wolf

**Comment by ecleipteon** [comment]
Hi @jonasthewolf ,

Sounds good, will do that thanks!

About the workshop in York. This is brilliant! We will also be at SSS'25, where I'll be giving a talk on the work I mentioned earlier. It’s exciting that we’ll be at the same conference—looking forward to meeting you and learning more about gsn2x!
